compiler-rt/lib/asan/asan_thread.cpp

```diff
@@ -93,6 +97,12 @@ AsanThreadContext *GetThreadContextByTidLocked(u32 tid) {
 AsanThread *AsanThread::Create(const void *start_data, uptr data_size,
                                u32 parent_tid, StackTrace *stack,
                                bool detached) {
+#if SANITIZER_WINDOWS
```

Comment:
This block can be moved before `AsanThread *t = AsanThread::Create(params, current_tid, &stack, detached);` in `INTERCEPTOR_WINAPI(HANDLE, CreateThread`